.changeset/purple-squids-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@compiled/eslint-plugin': patch
+---
+
+Add `onlyRunIfImportingCompiled` option to `jsx-pragma` rule, to specify additional libraries that should be considered Compiled imports
```

Comment:
Should this also mention `importSources`?

Reply:
whoops, good catch